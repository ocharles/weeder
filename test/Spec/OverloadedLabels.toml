roots = [ "Spec.OverloadedLabels.OverloadedLabels.root" ]

type-class-roots = false
