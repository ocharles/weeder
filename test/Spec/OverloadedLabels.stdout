main: test/Spec/OverloadedLabels/OverloadedLabels.hs:17:1: (Instance) :: Has Point "y" Int
